packages/integration/api-report.md

```diff
@@ -272,6 +284,18 @@ export type GoogleGcsIntegrationConfig = {
   privateKey?: string;
 };
 
+// @public
+export interface IGithubCredentialsProvider {
```

Comment:
We don't use `I` prefixes for interfaces, opting for just the direct name instead. Could be debated whether we should start doing that but I think it's best to hold off on that for now.

I think it might actually be possible to use the current class type as an interface type directly?

Reply:
Sure will do, Ill maybe call the old one DefaultGithubCredentialsProvider so that we are not confusing names?